prov/verbs/src/fi_verbs.c

```diff
@@ -2250,19 +2289,19 @@ fi_ibv_open_ep(struct fid_domain *domain, struct fi_info *info,
 		return -FI_EINVAL;
 
 	if (info->ep_attr) {
-		ret = fi_ibv_check_ep_attr(info->ep_attr, verbs_info);
+		ret = fi_ibv_check_ep_attr(info->ep_attr, _domain->info);
```

Comment:
Please rename variables that start with an underscore.  It looks like the name is already there, but go ahead and change it.